src/routes/safe/components/Balances/SendModal/screens/ModalHeader/style.ts

```diff
@@ -3,19 +3,23 @@ import { createStyles } from '@material-ui/core'
 
 export const styles = createStyles({
   heading: {
-    padding: `${md} ${lg}`,
+    padding: `${md} ${lg} 12px`,
```

Comment:
Also changed the bottom padding because it was disproportional somehow.

Reply:
Is the 'hover circle' still a circle?

Reply:
Yes
<img width="199" alt="Screenshot 2022-01-11 at 17 29 26" src="https://user-images.githubusercontent.com/381895/148982358-c08834c3-b3d9-4ff6-96df-f5fc407e59aa.png">